sim/hunter/talents.go

```diff
@@ -262,7 +262,7 @@ func (hunter *Hunter) applyPiercingShots() {
 		ActionID:    actionID,
 		SpellSchool: core.SpellSchoolPhysical,
 		ProcMask:    core.ProcMaskEmpty,
-		Flags:       core.SpellFlagNoOnCastComplete | core.SpellFlagIgnoreModifiers,
+		Flags:       core.SpellFlagNoOnCastComplete | core.SpellFlagIgnoreModifiers | core.SpellFlagIgnoreResists,
```

Comment:
I believe `SpellFlagIgnoreModifiers` already includes `SpellFlagIgnoreResists`.

Reply:
Nope. It's basically used for spells that snapshot other spells' damage (e.g. Blade Flurry, Sweeping Strikes, Necrosis), to prevent double dipping. SpellFlagIgnoreResists is now effectively a "SpellFlagBleeding", since there are no spells without (level-based) partial resists anymore.

Reply:
No haha, I mean this change is literally redundant: https://github.com/wowsims/wotlk/blob/8e734157beac1a64c1eb08b991d3ec39ea4f1c4e/sim/core/flags.go#L194

Reply:
I think it's for the bleed effect, will take a look.

Reply:
There are other effects which ignore resists. A good example would be mage Ignite and hunter Piercing Shots; these ignore resistances so that they don't double-dip on the resistance calculation.